includes/breadcrumb.inc

```diff
@@ -98,10 +99,15 @@ EOQ;
     $filter_map = function ($filter) {
       return "FILTER($filter)";
     };
-    $query_filters[] = "sameTerm(?collection_predicate, <fedora-rels-ext:isMemberOfCollection>) || sameTerm(?collection_predicate, <fedora-rels-ext:isMemberOf>)";
+    $compound_object_predicate = variable_get('islandora_compound_object_relationship', 'isConstituentOf');
```

Comment:
Sorry, managed to miss in my e-mails. I think we should absolutely go the hook route, even with recursion the invoke time is negligible as Drupal statically caches the hook implementations.